encoders/nlp/CLIPTextEncoder/__init__.py

```diff
@@ -17,21 +17,29 @@ class CLIPTextEncoder(BaseTorchEncoder):
     Internally, :class:`CLIPImageEncoder` wraps the `CLIP` model from https://github.com/openai/CLIP
 
     :param model_name: the name of the model. Supported models include ``ViT-B/32`` and ``RN50``.
+    :param args: additional positional arguments.
+    :param kwargs: additional positional arguments.
     """
     def __init__(self, model_name: str ='ViT-B/32',
                  *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.model_name = model_name
 
     def post_init(self):
+        """
+        Loads a model from clip specified in `model_name`. 
+        """
         import clip
         model, _ = clip.load(self.model_name, self.device)
         self.model = model
 
     @batching
     @as_ndarray
     def encode(self, data: 'np.ndarray', *args, **kwargs) -> 'np.ndarray':
-
+        """
+        Transforms a `np.ndarray` of strings of length `BatchSize` into
```

Comment:
Can you also add the params here (data, *args and **kwargs) and the return? otherwise it fails when we check for completeness of the docstrings

Reply:
updated